### ISSUE TITLE
CAMPD Home Page- 508 issue 

### ISSUE THREAD

**Issue by JanellC** [US-EPA-CAMD/PI-1#2290]
## Finding 

| Screen | Finding | WCAG 2.0 Standard(s) |
|---|---|---|
| Home Page | Skip nav is anchored past unique informational text on the home page https://campd-dev.app.cloud.gov/. It should be anchored right above the text ‘Your resource for emissions, compliance, and allowance data.’ | 2.4.1 |

## Context 

![image.png](https://images.zenhubusercontent.com/5f5928c1045cffdce7ce89c3/580bb9ef-9f77-44e2-9ed1-cca031d2f0b4)

**Comment by aprematta** [comment]
Tested on all pages, skip nav is functional and correctly anchored
